acknowledgments/index.md

```diff
@@ -13,6 +13,11 @@ served, and all of this infrastructure must be monitored and
 maintained.  On this page we'd like to thank the organizations and
 individuals that supply hardware for Void's use.
 
+## [23M GmbH](https://www.23m.com)
+
+23M Provides hosting for repo-de, our Tier 1 mirror located in central
```

Comment:
Maybe we should change `Provides hosting` to `sponsors a server`?

What do you think?